apps/frontend/src/components/questionary/questionaryComponents/GenericTemplate/QuestionaryComponentGenericTemplate.tsx

```diff
@@ -290,78 +293,90 @@ function QuestionaryComponentGenericTemplate(
+                          filter: {
+                            questionId: answer.question.id,
+                            proposalPk: state.proposal.primaryKey,
+                          },
+                        })
+                        .then((result) => {
+                          updateFieldValueAndState(
+                            result.genericTemplates,
+                            GENERIC_TEMPLATE_EVENT.ITEMS_MODIFIED
+                          );
+                        });
 
-                    setSelectedGenericTemplate(null);
-                  }}
-                  title={answer.question.question}
-                ></GenericTemplateContainer>
-              ) : (
-                <UOLoader />
-              )}
-            </StyledModal>
+                      setSelectedGenericTemplate(null);
+                    }}
+                    title={answer.question.question}
+                  ></GenericTemplateContainer>
+                ) : (
+                  <UOLoader />
+                )}
+              </DialogContent>
+            </StyledDialog>
             {openGenericTemplateSelectionOnCopyModal && (
-              <StyledModal
+              <StyledDialog
                 aria-labelledby="generic-template-selection-on-copy-modal-label"
                 aria-describedby="generic-template-selection-on-copy-modal"
                 open={openGenericTemplateSelectionOnCopyModal}
                 onClose={(): void =>
                   setOpenGenericTemplateSelectionOnCopyModal(false)
                 }
+                title={config.copyButtonLabel || 'Copy Sub Template'}
+                fullWidth
+                maxWidth="sm"
               >
-                <GenericTemplateSelectModalOnCopy
-                  close={(): void =>
-                    setOpenGenericTemplateSelectionOnCopyModal(false)
-                  }
-                  filter={{
-                    questionId: answerId,
-                  }}
-                  currentProposalPk={state.proposal.primaryKey}
-                  copyButtonLabel={config.copyButtonLabel}
-                  isMultipleCopySelect={config.isMultipleCopySelect || false}
-                  question={answer.question.question}
-                  handleGenericTemplateOnCopy={(
-                    copyAnswersInput: CopyAnswerInput[]
-                  ) => createGenericTemplateWithCopiedAnswers(copyAnswersInput)}
-                />
-              </StyledModal>
+                <DialogContent dividers>
+                  <GenericTemplateSelectModalOnCopy
+                    close={(): void =>
+                      setOpenGenericTemplateSelectionOnCopyModal(false)
+                    }
+                    filter={{
+                      questionId: answerId,
+                    }}
+                    currentProposalPk={state.proposal.primaryKey}
+                    isMultipleCopySelect={config.isMultipleCopySelect || false}
```

Comment:
we dont need that, since the new popup component expects title to be a string and it has been handled in https://github.com/UserOfficeProject/user-office-core/pull/695/files#diff-410db9e5255663d4b8817cb37dea74cee2c6ad2aca6a7df38ce55a432788564aR357

@janosbabik

Reply:
we dont need that, since the new popup component expects title to be a string and it has been handled in https://github.com/UserOfficeProject/user-office-core/pull/695/files#diff-410db9e5255663d4b8817cb37dea74cee2c6ad2aca6a7df38ce55a432788564aR357

@janosbabik